css-layout-classes/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>CSS Layout Classes</title>
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose your character!</h1>
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="Link" />
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-fourth">
+          <img src="images/kirby.png" alt="kirby" />
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/fox.png" alt="Fox" />
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png" alt="pikachu" />
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/ness.png" alt="ness" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third"></div>
+        <div>
+          <h2>Donkey Kong</h2>
+          <p>
+            Donkey Kong is a series of video games featuring the adventures of a
+            gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+            The franchise consists mainly of two game genres, but also includes
+            additional spin-off titles of various genres.
+          </p>
+          <p>
+            The games of the first genre are mostly single-screen
+            platform/action puzzle types, featuring Donkey Kong as the opponent
+            is an industrial construction setting.
+          </p>
+        </div>
+        <img src="images/dk.png" alt="Donkey Kong" />
+      </div>
```

Comment:
You have only one `column-one-third` here but it's on a `div` that is immediately closing? (line 47) This means that this column isn't doing anything. You have a wrapper `div` around you text and nothing around your `img`. You should wrap your `img` in a `div` and apply a column layout class to it. You have this right in the below section but you need to have it here too.